config/fieldparams/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@prysm//tools/go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "mainnet.go",
+        "minimal.go",  #keep
+    ],
```

Comment:
One thing that will have to change is we can no longer include both files in the build with `# keep` because of SSZ generation for the native beacon state. There is a "better" way to do this which selects one file based on the current config. You might want to incorporate it into your PR. If not, I will change it in my feature branch.
https://discord.com/channels/476244492043812875/550365651118850081/915341315661778946

Reply:
The Discord message has a bug, the second config should have `name = "mainnet"`